### PR TITLE
Hide old artifact links

### PR DESCRIPTION
**Pull request by Asizon**
Hide outdated artifact links when another PR workflow push.

image

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]
Like we have talked in the Slack chat, I think is better to include this as a step in the artifact-links.yml action. 
But I approve it because this only works with pull_request, until we find a workaround.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Pull request merged**
